server/views/pages/start.njk

```diff
@@ -1,13 +1,86 @@
 {% extends "../partials/prisonerBannerLayout.njk" %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
-
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
 
 {% set pageTitle = applicationName + " - Home" %}
 
+
+
 {% block content %}
     <h1>Start Page</h1>
     <div class="govuk-grid-row">
-        <div class="govuk-grid-column-full">
+
+        <div class="govuk-grid-column-two-thirds">
```

Comment:
not a reflection of what is in the prototype but an easy way to demo multiple appearances in a court case